test/e2e/specs/editor/blocks/pullquote.spec.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Quote', () => {
+	test.beforeEach( async ( { admin } ) => {
+		await admin.createNewPost();
+	} );
+
+	test( 'can be created by converting a quote and converted back to quote', async ( {
+		editor,
+		page,
+	} ) => {
+		await page.click( 'role=button[name="Add default block"i]' );
+
+		await page.keyboard.type( 'test' );
+		await editor.showBlockToolbar();
+
+		await page.click( 'role=button[name="Paragraph"i]' );
+
+		await page.click(
```

Comment:
Why do we have to use CSS selector here? I believe we can use role selector instead? In addition, since the test is not about the transforming flow, we might be able to just use `editor.transformBlockTo` instead.